src/components/Calculator.css

```diff
@@ -0,0 +1,68 @@
+#grid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-areas:
+    "result  result  result  result"
+    "mods    mods    mods    ops"
+    "nums    nums    nums    ops"
+    "nums    nums    nums    ops"
+    "nums    nums    nums    ops"
+    "nums    nums    nums    ops";
+  max-width: 80vw;
+}
```

Comment:
- [ ] [OPTIONAL] I think it could be good to reduce the width of your buttons because for now, they are so big 😉